### PR TITLE
fix(docu): change link of deployment description

### PR DESCRIPTION
**Pull request by Elweyn**
## 🍰 Pullrequest
As developer I want to see the documentation of the deployment.

### Issues
- None

### Todo

- [X] Link to deployment README.md
- [X] Bare-metal installation on root

### CODE REVIEW

File: deployment/README.md [hunk 1]
```diff
@@ -74,42 +83,50 @@ vi /etc/nginx/http.d/admin.conf
...
 
-Configure backend connection:
+### Configure backend connection
+
 ```bash
 cp presenter/.env.dist presenter/.env
 # adjust the .env config as needed
 vi presenter/.env
 # PUBLIC_ENV__ENDPOINTS__GRAPHQL_URI=http://localhost/api
+```
```
Comment (Mogge) [inline deployment/README.md:117]: This must be done for frotend and admin as well

**Review by Mogge — CHANGES_REQUESTED** [review]
Add frontend and admin as well to update backend url

File: deployment/README.md [hunk 2]
```diff
@@ -74,42 +83,66 @@ vi /etc/nginx/http.d/admin.conf
...
+
 ```bash
 cp backend/.env.dist backend/.env
 # adjust the .env config as needed
 vi backend/.env
 # DATABASE_URL="mysql://dreammall:SECRET@localhost:3306/dreammall.earth"
 ```
 
-Configure backend connection:
+### Configure backend connection
```
Comment (Mogge) [inline deployment/README.md:110]:
```suggestion
### Configure backend connections

#### Presenter
```

File: deployment/README.md [hunk 3]
```diff
@@ -74,42 +83,66 @@ vi /etc/nginx/http.d/admin.conf
...
+
 ```bash
 cp presenter/.env.dist presenter/.env
 # adjust the .env config as needed
 vi presenter/.env
 # PUBLIC_ENV__ENDPOINTS__GRAPHQL_URI=http://localhost/api
+```
+
+### Configure frontend connection
```
Comment (Mogge) [inline deployment/README.md:119]:
```suggestion
#### Frontend
```

File: deployment/README.md [hunk 4]
```diff
@@ -74,42 +83,66 @@ vi /etc/nginx/http.d/admin.conf
...
+
+```bash
+cp frontend/.env.dist frontend/.env
+# adjust the .env config as needed
+vi frontend/.env
+```
+
+### Configure admin connection
```
Comment (Mogge) [inline deployment/README.md:127]:
```suggestion
#### Admin
```

**Review by Mogge — CHANGES_REQUESTED** [review]

**Review by mahula — APPROVED** [review]
Good job!

**Pull request merged**